trunk/src/app/srs_app_server.cpp

```diff
@@ -43,6 +43,7 @@ using namespace std;
 #ifdef SRS_GB28181
 #include <srs_app_gb28181.hpp>
 #endif
+#include <srs_app_rtc_server.hpp>
```

Comment:
Maybe add conditional compile here: `#ifdef SRS_RTC`, or consider move the `extern SrsResourceManager _srs_rtc_manager;` declaration and initialization to `srs_app_rtc_network.hpp|cpp`.

Reply:
Fixed